api/src/main/java/io/lumeer/engine/api/data/DataStorageStats.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * -----------------------------------------------------------------------\
+ * Lumeer
+ *  
+ * Copyright (C) since 2016 the original author or authors.
+ *  
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------/
+ */
+package io.lumeer.engine.api.data;
+
+/**
+ * Carries statistics about database usage.
+ *
+ * @author <a href="mailto:marvenec@gmail.com">Martin Večeřa</a>
+ */
+public class DataStorageStats {
+
+   private String databaseName;
+
+   private long collections;
+
+   private long documents;
+
+   private long dataSize;
+
+   private long storageSize;
```

Comment:
It would be useful to add a JavaDoc explaining these attributes. Especially the difference between `dataSize` and `storageSize` is not obvious unless you read MongoDB documentation.